receiver/splunkhecrawreceiver/README.md

```diff
@@ -0,0 +1,45 @@
+# Splunk HEC Raw Receiver
+
+The Splunk HEC raw receiver accepts data delimited by newline separators and ingests them as logs.
```

Comment:
Is there anything to link to describing the HEC raw protocol?

Reply:
added a link